examples/Planning/ForecastReach.php

```diff
@@ -0,0 +1,389 @@
+<?php
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Ads\GoogleAds\Examples\Planning;
+
+require __DIR__ . '/../../vendor/autoload.php';
+
+use GetOpt\GetOpt;
+use Google\Ads\GoogleAds\Examples\Utils\ArgumentNames;
+use Google\Ads\GoogleAds\Examples\Utils\ArgumentParser;
+use Google\Ads\GoogleAds\Lib\V3\GoogleAdsClient;
+use Google\Ads\GoogleAds\Lib\V3\GoogleAdsClientBuilder;
+use Google\Ads\GoogleAds\Lib\OAuth2TokenBuilder;
+use Google\Ads\GoogleAds\Lib\V3\GoogleAdsException;
+use Google\Ads\GoogleAds\V3\Common\DeviceInfo;
+use Google\Ads\GoogleAds\V3\Common\GenderInfo;
+use Google\Ads\GoogleAds\V3\Enums\DeviceEnum\Device;
+use Google\Ads\GoogleAds\V3\Enums\GenderTypeEnum\GenderType;
+use Google\Ads\GoogleAds\V3\Enums\ReachPlanAdLengthEnum\ReachPlanAdLength;
+use Google\Ads\GoogleAds\V3\Enums\ReachPlanAgeRangeEnum\ReachPlanAgeRange;
+use Google\Ads\GoogleAds\V3\Errors\GoogleAdsError;
+use Google\Ads\GoogleAds\V3\Services\CampaignDuration;
+use Google\Ads\GoogleAds\V3\Services\PlannableLocation;
+use Google\Ads\GoogleAds\V3\Services\PlannedProduct;
+use Google\Ads\GoogleAds\V3\Services\Preferences;
+use Google\Ads\GoogleAds\V3\Services\ProductAllocation;
+use Google\Ads\GoogleAds\V3\Services\ProductMetadata;
+use Google\Ads\GoogleAds\V3\Services\ReachForecast;
+use Google\Ads\GoogleAds\V3\Services\Targeting;
+use Google\ApiCore\ApiException;
+use Google\Protobuf\BoolValue;
+use Google\Protobuf\Int32Value;
+use Google\Protobuf\Int64Value;
+use Google\Protobuf\StringValue;
+
+/**
+ * This example demonstrates how to interact with the ReachPlanService to find plannable
+ * locations and product codes, build a media plan, and generate a video ads reach forecast.
+ */
+class ForecastReach
+{
+    const CUSTOMER_ID = 'INSERT_CUSTOMER_ID_HERE';
+
+    // US
```

Comment:
Could you check if the comment at the end of the line violates PSR12?
If not, I think doing as follow is more concise and readable.

private const LOCATION_ID = '2840'; // US

(Note that private modifier is needed to comply with PSR12)

Reply:
It does not, changing for it.

`private` only fixes a warning in PSR-12. I think it is better to keep it consistent with all existing code examples (do not use `private`). WDYT of fixing this in all code examples at the same time in a separate PR instead?

Reply:
SG, will do for now on